<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RigLab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="uiBanner_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 bold italic 18pt &quot;Trebuchet MS&quot;;
color: rgb(220, 220, 220);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 20, 20, 255), stop:1 rgba(40, 40, 40, 255));</string>
     </property>
     <property name="text">
      <string>Naming Convention Editor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="content_layout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QTabWidget" name="contents_tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="rules">
        <attribute name="title">
         <string>Rules</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="rules_vLayout">
           <item>
            <widget class="QListWidget" name="rules_listWidget"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="rules_hLayout">
             <property name="spacing">
              <number>1</number>
             </property>
             <item>
              <widget class="QToolButton" name="addRule_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="removeRule_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tokens">
        <attribute name="title">
         <string>Tokens</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="tokens_vLayout">
           <item>
            <widget class="QListWidget" name="tokens_listWidget"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="tokens_hLayout">
             <property name="spacing">
              <number>1</number>
             </property>
             <item>
              <widget class="QToolButton" name="addToken_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="removeToken_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="properties_groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Properties</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="ruleProperties_frame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="expression_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Expression</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="expression_hLayout">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="expression_lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="expression_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="tokenProperties_frame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="values_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Values</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="values_hLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QTableWidget" name="values_tableWidget">
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="gridStyle">
                <enum>Qt::DotLine</enum>
               </property>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>103</number>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>19</number>
               </attribute>
               <column>
                <property name="text">
                 <string>Key</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Value</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="values_vLayout">
               <property name="spacing">
                <number>1</number>
               </property>
               <item>
                <widget class="QToolButton" name="addValue_button">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="removeValue_button">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="values_spacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="default_label">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="default_comboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="properties_frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="description_label">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextEdit" name="description_textEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>contents_tabWidget</tabstop>
  <tabstop>rules_listWidget</tabstop>
  <tabstop>addRule_button</tabstop>
  <tabstop>removeRule_button</tabstop>
  <tabstop>tokens_listWidget</tabstop>
  <tabstop>addToken_button</tabstop>
  <tabstop>removeToken_button</tabstop>
  <tabstop>expression_lineEdit</tabstop>
  <tabstop>expression_button</tabstop>
  <tabstop>addValue_button</tabstop>
  <tabstop>removeValue_button</tabstop>
  <tabstop>default_comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>contents_tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>contents_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>contents_changed()</slot>
 </slots>
</ui>
